src/Confluent.Kafka/Serialization/StringSerializer.cs

```diff
@@ -26,32 +30,94 @@ public class StringSerializer : ISerializer<string>
         private Encoding encoding;
 
         /// <summary>
+        ///     Name of the encoding configuration parameter for the case of key serialization.
```

Comment:
ok, i'll reword this (but will avoid use of the word 'type' too, as that's kind of overloaded in this context).